tests/core/custom-device-reg-extras/task.yaml

```diff
@@ -3,8 +3,9 @@ summary: |
     with the prepare-device gadget hook and this can set request headers,
     a proposed serial and the body of the serial assertion
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*-64]
+# TODO:UC20: enable for UC20, it assumes /var/lib/snapd/seed/assertions/model
+#            which we don't have currently
+systems: [ubuntu-core-1*]
```

Comment:
Please keep the previous regex because the new one will match with some arm devices ubuntu-core-16-arm-64/32 which will fail because the model used on the test is done for pc. Also is gonna match with ubuntu-core-16-32 which is gonna fail as well.